lib/delayed/backend/iron_mq_backend.rb

```diff
@@ -1,18 +1,18 @@
 require_relative 'iron_mq_config'
 
 module Delayed
-  class Worker
+  class IronMqBackend
     class << self
       attr_accessor :config, :ironmq, :queue_name, :delay, :timeout, :expires_in,
```

Comment:
`attr_accessor : queue_name` is a default queue, `default_queue` seems better name